update-asdf-and-dockerfile/action.yml

```diff
@@ -95,14 +95,15 @@ runs:
         cat package.json | jq ".engines.node |= \"^$(node --version | tr -d v)\"" | jq ".engines.npm |= \"^$(npm --version)\"" > updated-package.json && \
         mv updated-package.json package.json && \
         echo "package.json engines after update:" && \
-        jq ".engines" package.json
+        jq ".engines" package.json && \
+        npm install
```

Comment:
Hmmm or I think we can alter the change of the script to only trigger this part if the node version is indeed going to be updated and then the npm install would only happen in that case 👍

Reply:
Tried updating it to do that. I don't know how to test this locally though, so I'm not 100% sure if the syntax is correct?

Reply:
> I don't know how to test this locally though
Me neither but the worst thing this will do is to create wrong content in the PR so I think we can just merge and do a test run 👍

Reply:
It worked 🥳 Also didn't take that long (1min 40s vs 1min 27sec)

Reply:
Should we still change to this just to save the unneccessary bandwidth? https://stackoverflow.com/a/57867531